test/runner/browserstack/api.js

```diff
@@ -0,0 +1,332 @@
+/**
+ * Browserstack API is documented at
+ * https://github.com/browserstack/api
+ */
```

Comment:
If we don't plan to enable BrowserStack support here, maybe let's drop it? We're not keeping it in UI either.

Reply:
I'd prefer to leave it in. It's a little closer to core & migrate and at least allows for local browserstack testing. I was actually planning on aligning UI with color after this, not just to add browserstack, but to make all 4 as close as possible.

Reply:
OK, fairly enough.

That said, with almost the same code duplicated in 4 places now, I really think we should extract this to its own repo & an npm package.

Reply:
Yeah, I'm leaning towards that as well, but it will need to come later.

Reply:
Definitely something to be done separately later. I hope you can find some time to get that done!